src/bci_build/package/python/README.md.j2

```diff
@@ -0,0 +1,63 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Python](https://www.python.org/) is an interpreted, interactive, object-oriented, open-source programming language. It incorporates modules, exceptions, dynamic typing, very high-level dynamic data types, and classes. Python combines remarkable power with clear syntax. It has interfaces to many system calls and libraries, as well as to various window systems, and is extensible in C or C++. It is also usable as an extension language for applications that need a programmable interface.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+WORKDIR /app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD [ "python", "./main-script.py" ]
+```
+
+Build and run the container image:
+
+```ShellSession
+$ podman build -t my-python-app .
+$ podman run -it --rm my-python-app
+```
+
+To run a single script inside a container, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} python3 script.py
+```
+
+To run the tests with [pytest](https://pytest.org) inside a container, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} pytest -vv
```

Comment:
yes, true, would be better if I replace it with unittest or add a note that pytest must be installed? or I can also drop this line entirely.

Reply:
I added a note that pytest should be installed.

Reply:
I've had another thought about this and I think we should remove this direct pytest invocation. Imho this will only work reliably in the following scenarios:
1. you build a new container into which you `pip install pytest` and use the new container instead of bci/python
2. you run the usual `python -m venv .env3 && source .env3/bin/activate && pip install pytest` **but** with the project root bind mounted into the container so that the virtualenv on your machine is created using the container's python.

I find 1. to be a bad advertisement, 2. just far too brittle in practice and overall I have my doubts whether this is a common use case or whether most users wouldn't just run this as part of the container build process.

Reply:
We can remove it, no problem, there's an example of how to call a single script, most people would know that can invoke tox, pytest, unitest, or else similarly.